pymatviz/_colors.py

```diff
@@ -0,0 +1,256 @@
+"""Colors used in pymatviz."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Union
+
+
+if TYPE_CHECKING:
+    from typing import Final
+
+Color = Union[tuple[float, float, float], tuple[float, float, float, float]]
+
+# Element type based colors
+ELEM_TYPE_COLORS: Final = {
+    "Diatomic Nonmetal": "green",
+    "Noble Gas": "purple",
+    "Alkali Metal": "red",
+    "Alkaline Earth Metal": "orange",
+    "Metalloid": "darkgreen",
+    "Polyatomic Nonmetal": "teal",
+    "Transition Metal": "blue",
+    "Post Transition Metal": "cyan",
+    "Lanthanide": "brown",
+    "Actinide": "gray",
+    "Nonmetal": "green",
+    "Halogen": "teal",
+    "Metal": "lightblue",
+    "Alkaline Metal": "magenta",
+    "Transactinide": "olive",
+}
+
+# The following element-based colors are taken from elementari:
+# https://github.com/janosh/elementari
+ELEM_COLORS_JMOL: dict[str, Color] = {
+    "H": (255, 255, 255),
```

Comment:
Need double check, `matplotlib` color in 0-1 range or 0-255?

Reply:
they're 0-1:

```py
from matplotlib.colors import rgb2hex
from matplotlib.typing import ColorType

ColorType
rgb2hex((1,1,1))
```

```py
rgb2hex((255,1,1))
>>> ValueError: RGBA values should be within 0-1 range
```

Reply:
i think the `colors` should really be public. i can see use cases where people want to import the JMOL or VESTA element color schemes